### ISSUE TITLE
Launcher lags while using the Pixel 2 Live wallpapers

### ISSUE THREAD

**Issue by kuberrohilla** [amirzaidi/Launcher3#277]
When using the pixel's live wallpapers, the app drawer lags a lot. Both when you pull up for app drawer and  the scrolling in the app drawer lags. 

Pixel 2 Live wallpaper app - https://www.apkmirror.com/apk/pranav-pandey/pixel-wallpapers-2017-port/pixel-wallpapers-2017-port-8-0-0-pp-6-0-arm64-release/pixel-wallpapers-2017-port-8-0-0-pp-6-0-arm64-android-apk-download/ (This is integrated with google's Wallpaper app).

Read more about this app  on Android Authority here - http://www.androidpolice.com/2017/10/16/get-new-pixel-2-live-wallpapers-android-device-apk-download/

Device- Mi A1
Android- v7.1.2
ROM-Stock ROM
Launcher version - latest

**Comment by SwarfegaGit** [comment]
Works fine for me.

Nexus 6P 8.0

**Comment by nikkomercado** [comment]
It isn't the launcher. It lags because of your phone's hardware limitations. Just like mine.

**Comment by TPS** [comment]
For those w/ slower devices, maybe y'all want to try turning off/decreasing animation settings in Settings » Developer Options?

**Comment by kuberrohilla** [comment]
It's already set for 0.5x and is SD625 considered a slow SOC?

**Comment by mlleemiles** [comment]
Works fine for me.

Z5 Compact

**Comment by TPS** [comment]
> It's already set for 0.5x and is SD625 considered a slow SOC?

@kuberrohilla If turning it off solves the problem, then, evidently, yes. 😶🤷

**Comment by kuberrohilla** [comment]
@TPS No turning it off didn't resolve the problem.

**Comment by dcshoecomp** [comment]
@kuberrohilla 
> It's already set for 0.5x and is SD625 considered a slow SOC?

According to gsmarena's basemark os ii score you are no where near the specs of the nexus 6 or original pixel. So you must be light years behind the pixel 2.

![image](https://user-images.githubusercontent.com/16295807/31819829-7f8cca50-b56c-11e7-988e-becb97b918b4.png)

**Comment by kuberrohilla** [comment]
Anyone who is running this launcher on non-SD8xx chip and can confirm if there is any such issue with pixel 2 live wallpapers? That way we could say that it's a hardware limitation. But i don't think that should be an issue as SD625 is a mid range processor.

**Comment by RikKoedoot** [comment]
@kuberrohilla I have a Moto g4 plus and for me it also lags

**Comment by djassal** [comment]
I also have Mi A1 and when i remove the live wallpaper it works fine.
But with live wallpapers it lags.
If Anyone know a better live wallpaper app please attach link

**Comment by Vengance09** [comment]
I have an LG G3 with the 801 and they lag for me. Just went back to static ones. Plus the hit to the battery is to much.

**Comment by RamenSayami** [comment]
used to run this app on mi4(sd 801) it worked fine...
now i got mi note 5(sd 625) i miss these live wallpapers... lag lag lag :/ im sure its hardwarw issue.. mayb coz its got older openVG version.. got smth to do with 2d gpu rendering for ui :/ hopin for a fix tho

**Comment by RikKoedoot** [comment]
It's probably your device?

**Comment by RamenSayami** [comment]
yeah it kinda is... but isnt there a way to optimize or stuff like that so that it runs on snapdragon 625? i thot 625 was better than 801 with 64 bit processing and better gpu support.. how is it not able to run live wallpaper smoothly?

**Comment by RikKoedoot** [comment]
It also depends on the wallpaper you use

**Comment by RikKoedoot** [comment]
But this isn't launcher fault

**Comment by arghyadip1999** [comment]
This is neither the launcher's fault nor the processor's. The wallpapers lag because of the UI. I had installed the Pixel Experience custom ROM on my Redmi 3s(which in case you don't know has a Snapdragon 430 chipset ). I use the Pixel 2 live wallpapers on it and it is butter smooth. Not even a single frame drop. However when I used to run it on MIUI it lagged like hell. Also it seems that if you have the Pixel launcher or the Google Now launcher (not Android one launcher) installed as a system app like in Pixels and Nexus devices, there is no lag. However 3rd party launcher's face a lag issue.
